docs/luigi-client-api.md

```diff
@@ -335,6 +335,22 @@ LuigiClient.linkManager().fromClosestContext().navigate('/users/groups/stakehold
 
 Returns **[linkManager](#linkmanager)** link manager instance
 
+#### fromVirtualTreeRoot
+
+Sets the current navigation base to the parent node that is defined as virtualTree. This method works only when already inside a virtualTree.
+
+##### Examples
+
+```javascript
+LuigiClient.linkManager().fromVirtualTreeRoot().navigate('/users/groups/stakeholders')
+```
+
+Returns **[linkManager](#linkmanager)** link manager instance
+
+**Meta**
+
+-   **since**: NEXTRELEASE
```

Comment:
```suggestion
```
Is it supposed to be displayed as `nextrelease` in the docu? Or will it automatically update with the next release? It doesn't seem ideal to document something if it wasn't released yet.

Reply:
Thats why it has NEXTRELEASE and no version number. It implicitly tells - upcoming.